packages/magento-product-configurable/components/ConfigurableProductReviewChip/ConfigurableProductReviewChip.tsx

```diff
@@ -0,0 +1,23 @@
+import { ProductReviewChip } from '@graphcommerce/magento-review'
+import { useConfigurableOptionsSelection } from '../../hooks'
+
+export type ProductReviewChipProps = {
+  url_key?: string | null
+  rating_summary?: number
+  index?: number
+}
+
+export function ConfigurableProductReviewChip(props: ProductReviewChipProps) {
+  const { rating_summary, url_key, index = 0 } = props
+  const { configured } = useConfigurableOptionsSelection({ url_key, index })
+
+  const simpleProductRatingSummary =
+    configured?.configurable_product_options_selection?.variant?.rating_summary
+
+  return (
+    <ProductReviewChip
+      rating={simpleProductRatingSummary ?? rating_summary}
+      reviewSectionId='reviews'
+    />
+  )
+}
```

Comment:
Interesting component, not sure what to do with this? Should we always try and get the reviews from the configurable or does it make sense to get the reviews from the variant as well?

Reply:
Currently, all the reviews are always placed with the configurable product. This means that there are more reviews available to show. However, if the variants are very different from each other, it might be better to allow placing a separate review for each variant. I haven't decided yet on the best way to make this happen, maybe it could be an optional feature. What do you think?